src/commands/release/import.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2016-2024 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Flags, Args } from '@oclif/core';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, stripIndent } from '../../utils/lazy';
+import { apply } from '@balena/release-bundle';
+import { createReadStream } from 'fs';
+import { ExpectedError } from '../../errors';
+
+export default class ReleaseImportCmd extends Command {
+	public static description = stripIndent`
+		Imports a release from a file to an application or fleet.
+
+		To save a release into a file, use 'balena release export'.
+
+		Use the --override-version option to specify the version 
+		of the imported release, overriding the one saved in the file.
+`;
+	public static examples = [
+		'$ balena release import ../path/to/release.tar -f 1234567',
+		'$ balena release import ../path/to/release.tar -f myFleet',
+		'$ balena release import ../path/to/release.tar -f myOrg/myFleet',
+		'$ balena release import ../path/to/release.tar -f myOrg/myFleet -V 1.2.3',
+	];
+
+	public static usage = 'release import <file>';
+
+	public static flags = {
+		fleet: { ...cf.fleet, exclusive: ['device'] },
+		'override-version': Flags.string({
+			description:
+				'Imports this release with the specified version overriding the version in the file.',
+			char: 'V',
```

Comment:
I'm hesitant to include a char for the command at the moment, as we need to discuss and figure out how to determine what char to use for commands, especially without creating confusion/overlap with out commands

WDYT @otaviojacobi , would you also skip the short version and remove `char` for now?

Reply:
I am fine with it if there is a strong reason to have it, but I would also prefer not to have it because it is very easily to mistake with the much more common `v` char we already have for several commands which serves for verbose

Reply:
We just need an option to override the version.  I think we could do without using a character to represent the option.

Reply:
I removed the character flag for the option.  Users will just need to use `--override-version` to use a different version than the one that the release have.